docs/framework/network-programming/tls.md

```diff
@@ -61,10 +61,6 @@ For Windows Forms and other applications, see [How to: Target a Version of the .
 
 Use the following sections to verify you're not using a specific TLS or SSL version.
 
-## If your app targets .NET Framework 4.7 or later versions
```

Comment:
if you remove this parent section, then you need to adjust the children headings accordingly to be H2 instead of H3

Reply:
I wanted to roll them intentionally into subsections of the previous section "Audit your code and make code changes".
If that is violating some rules, we will have to come up with similar title as replacement then -- something like "For all NET apps" ... which seems weird IMO. I'm open to suggestions here.

Reply:
Ah cool. Then it's fine like this.